python/ray/tests/test_resource_demand_scheduler.py

```diff
@@ -307,7 +307,7 @@ def testScaleUpIgnoreUsed(self):
             head_ip, {
                 "CPU": 4,
                 "GPU": 1
-            }, {"GPU": 1}, {},
+            }, {"GPU": 0}, {},
```

Comment:
Can we add some more unit tests for this?

Reply:
not sure what else to be tested? the issue with the test was that i put the incorrect value here.

Reply:
You can test a couple more edge cases such as multiple nodes, and also what happens if you have missing ips, extra ips, etc. The current test is fairly limited.